docs/troubleshooting-guide/cant-send-email.md

```diff
@@ -0,0 +1,61 @@
+# Metabase isn't sending email
+
+You have told Metabase to send email notifications, but:
+
+- the notifications aren't arriving.
+
+## Are the email credentials correct?
+
+**Root cause:** The email address and/or password have been set up incorrectly.
+
+**Steps to take:**
+
+1. In the Admin Panel, select "Email Settings" and click "Send test email"
+2. Verify that the email is delivered to the test account.
+3. If the message is not sent or an error message is displayed in Metabase, try to use the same account credentials in another email program and see if they work. If they do, you may have found a---please [report it][bugs].
+
+## Is the HOSTNAME of the Metabase server set correctly?
+
+**Root cause:** Some email delivery services, such as GMail, will report errors when the mail sender's HOSTNAME doesn't match the actual host's name. In particular, by default EC2 instances of Metabase set the hostname to the local IP address unless configured otherwise.
+
+**Steps to take:**
+
+1. Make sure that the HOSTNAME is being set correctly. FIXME where and how to do this? It doesn't seem to be in our docs.
+
```

Comment:
I have never seen this reported. I think this might have been confused with something else.
```suggestion
```